### DISCUSSION TITLE
Bespoke Model Support

### DISCUSSION THREAD

**Discussion by uckthis** [SYSTRAN/faster-whisper#450] [Ideas]
So far it's the best module I have used with Whisper. I use Faster with Subtitle Edit and I think the next logical upgrade would be support for bespoke models. There are so many models trained on different languages which are better than the base models. Hugging Face is full of French, German, Italian and so many other models which we could use with Faster. I have zero computer skills, so I don't know if it's possible with Faster Subtitle Edit Module.

Thank you.

**Comment by guillaumekln** [comment]
Do you mean fine-tuned models? If yes, these are already supported after a conversion: https://github.com/guillaumekln/faster-whisper#model-conversion.

I don't know about the integration in Subtitle Edit though.

**Comment by uckthis** [comment]
Yes, I meant fine-tuned models. There's a UI in Subtitle Edit, and it'll be a lot easier to do this in a UI than CMD for someone technically challenged like myself.
